### PR TITLE
bugfix: added enter key support for register/login page

### PR DESCRIPTION
**Pull request by AlexOla-NG**

### CODE REVIEW

**Comment by AlexOla-NG** [comment]
This is the PR. Please approve

File: frontend/.vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "typescript.tsdk": "node_modules/.pnpm/typescript@4.9.5/node_modules/typescript/lib",
+  "typescript.tsdk": "node_modules\\typescript\\lib",
```
Comment (puckzxz) [inline frontend/.vscode/settings.json:2]: Could we revert this change? We don't need this and checking out locally just overwrites this with the old value

**Review by puckzxz — CHANGES_REQUESTED** [review]
Could we also remove the `package-lock.json` too? We're using `pnpm` for this project which creates its own lock file. @ImVexed and I are currently working on the contributing section of the README so sorry for any confusion the lack of one may have caused.

Overall though this works great! Thanks!

Once the 2 small comments are addressed then this will be good to merge 🚀

File: frontend/.vscode/settings.json (hunk 1 above)
Comment (AlexOla-NG) [reply frontend/.vscode/settings.json]: Hi @puckzxz, apologies for the late response.

I've made the changes you requested (removed package-lock & reverted changes on the the typescript.tsdk file). Please confirm🙏🏾

**Review by puckzxz — APPROVED** [review]
🚀

**Comment by AlexOla-NG** [comment]
🔥🔥🔥
